Microsoft.Toolkit.Uwp.UI.Controls/TokenizingTextBox/TokenizingTextBoxItem.xaml

```diff
@@ -0,0 +1,105 @@
+    <Style x:Key="TokenizingTextBoxItemClearButtonStyle" TargetType="Button">
+        <Setter Property="Visibility" Value="Visible" />
+        <Setter Property="Background" Value="Transparent" />
+        <Setter Property="BorderThickness" Value="0" />
+    </Style>
+
+    <!-- Default style for TokenizingTextBoxItem -->
+    <Style TargetType="controls:TokenizingTextBoxItem">
+        <Setter Property="Background" Value="{ThemeResource SystemControlTransparentBrush}" />
+        <Setter Property="BorderBrush" Value="{ThemeResource SystemControlTransparentBrush}" />
+        <Setter Property="BorderThickness" Value="{StaticResource TokenizingTextBoxItemBorderThickness}" />
+        <Setter Property="ClearButtonStyle" Value="{StaticResource TokenizingTextBoxItemClearButtonStyle}" />
+        <Setter Property="HorizontalContentAlignment" Value="{StaticResource TokenizingTextBoxItemHorizontalContentAlignment}" />
+        <Setter Property="MinHeight" Value="{StaticResource TokenizingTextBoxItemMinHeight}" />
+        <Setter Property="Padding" Value="{StaticResource TokenizingTextBoxItemPadding}" />
+        <Setter Property="VerticalContentAlignment" Value="{StaticResource TokenizingTextBoxItemVerticalContentAlignment}" />
+        <Setter Property="IsTabStop" Value="True" />
+        <Setter Property="UseSystemFocusVisuals" Value="True" />
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="controls:TokenizingTextBoxItem">
+                    <Grid 
+                        Name="RootPanel"
+                        Background="{TemplateBinding Background}"
+                        BorderBrush="{TemplateBinding BorderBrush}"
+                        BorderThickness="{TemplateBinding BorderThickness}"
+                        Padding="{TemplateBinding Padding}">
+
+                        <Rectangle 
+                            Name="SelectedState"
+                            Visibility="Collapsed"
+                            Fill="{ThemeResource SystemControlHighlightListAccentLowBrush}" />
+
+                        <StackPanel Orientation="Horizontal">
+                            <ContentPresenter
+                                Name="ContentPresenter"
+                                HorizontalContentAlignment="{TemplateBinding HorizontalContentAlignment}"
+                                VerticalContentAlignment="{TemplateBinding VerticalContentAlignment}"
+                                Content="{TemplateBinding Content}"
+                                ContentTemplate="{TemplateBinding ContentTemplate}" />
+
+                            <Button 
+                                Name="PART_ClearButton"
+                                Width="{TemplateBinding MinHeight}"
+                                Height="{TemplateBinding MinHeight}"
+                                IsTabStop="False"
+                                Style="{TemplateBinding ClearButtonStyle}">
```

Comment:
You can add automation property for this for the localization via x:Uid, see the TabView close button as an example.

Reply:
What localization? The "Clear" string corresponds to the Symbols Enum value for the button icon.